examples/00-EDB/09_Configuration.py

```diff
@@ -0,0 +1,165 @@
+comp2 = edbapp.core_components.components["C3A3"]
+comp2.model_type, comp2.value
+
+###############################################################################
+# Check Component Definition
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+edbapp.core_components.nport_comp_definition
+
+###############################################################################
+# Save Edb
+# ~~~~~~~~
+edbapp.save_edb()
+
+###############################################################################
+# Configure Setup
+# ~~~~~~~~~~~~~~~
+# This step allows to define the project. It includes:
+#  - Definition of nets to be included into the cutout
+#  - Cutout details
+#  - Components on which to create the ports
+#  - Simulation settings
+
+sim_setup = SimulationConfiguration()
+sim_setup.solver_type = SolverType.SiwaveSYZ
+sim_setup.cutout_subdesign_expansion = 0.01
+sim_setup.do_cutout_subdesign = True
+sim_setup.signal_nets = ["PCIE0_RX0_P", "PCIE0_RX0_N", "PCIE0_TX0_P_C", "PCIE0_TX0_N_C", "PCIE0_TX0_P", "PCIE0_TX0_N"]
+sim_setup.components = ["U2A5", "J2L1"]
+sim_setup.power_nets = ["GND"]
+sim_setup.start_frequency = "100Hz"
+sim_setup.stop_freq = "6GHz"
+sim_setup.step_freq = "10MHz"
+
+###############################################################################
+# Run Setup
+# ~~~~~~~~~
+# This step allows to create the cutout and apply all settings.
+
+sim_setup.export_json(os.path.join(project_path, "configuration.json"))
+edbapp.build_simulation_project(sim_setup)
+
+###############################################################################
+# Plot Cutout
+# ~~~~~~~~~~~
+# Plot cutout once finished.
+
+edbapp.core_nets.plot(None,None)
+
+###############################################################################
+# Save and Close EDB
+# ~~~~~~~~~~~~~~~~~~
+# Edb will be saved and closed in order to be opened by Hfss 3D Layout and solved.
+
+edbapp.save_edb()
+edbapp.close_edb()
+
+###############################################################################
+# Open Aedt
+# ~~~~~~~~~
+# Project folder aedb will be opened in AEDT Hfss3DLayout and loaded.
+h3d = Hfss3dLayout(specified_version="2022.2", projectname=target_aedb, non_graphical=non_graphical)
+
+###############################################################################
+# Analyze
+# ~~~~~~~
+# Project will be solved.
+h3d.analyze_nominal()
+
+###############################################################################
+# Get Results
+# ~~~~~~~~~~~
+# S Parameter data will be loaded at the end of simulation.
+solutions = h3d.post.get_solution_data()
+
+###############################################################################
+# Plot Results
+# ~~~~~~~~~~~~
+# S Parameter data will be plotted at the end of simulation.
+solutions.plot(solutions.expressions, "db20")
+
+###############################################################################
+# Save and Close AEDT
+# ~~~~~~~~~~~~~~~~~~~
+# S Parameter data will be plotted at the end of simulation.
```

Comment:
I think we could change this comment because the S-Parameters were displayed in the previous step.